src/parser/shared/modules/features/TimelineTab/TabComponent/SpellTimeline.js

```diff
@@ -193,18 +193,26 @@ class SpellTimeline extends React.PureComponent {
               })}
             </div>
           )}
-          {this.spells.map(spellId => (
-            <SpellRow
-              key={spellId}
-              className="lane"
-              events={historyBySpellId[spellId] || []}
-              buffEvents={buffEvents[abilities.getBuffSpellId(spellId)]}
-              start={start}
-              totalWidth={totalWidth}
-              secondWidth={secondWidth}
-              showCooldowns={showCooldowns}
-            />
-          ))}
+          {this.spells.map(spellId => {
+            let combinedBuffEvents = [];
+            if (abilities.getBuffSpellId(spellId) && abilities.getBuffSpellId(spellId) instanceof Array) {
```

Comment:
In order to not call the thing all the time (since afaik, it iterates through Abilities every single time), I'd suggest something like this:
```javascript
let combinedBuffEvents = [];
const buffSpellId = abilities.getBuffSpellId(spellId);
if (buffSpellId && buffSpellId instanceof Array) {
  buffSpellId.forEach(spell => {
    combinedBuffEvents.push(buffEvents[spell]);
  });
} else if (buffSpellId) {
  combinedBuffEvents = buffEvents[buffSpellId];
}
```